stdlib/stdlib.mli

```diff
@@ -241,6 +241,14 @@ external ( @@ ) : ('a -> 'b) -> 'a -> 'b = "%apply"
    @since 4.01
 *)
 
+val ( % ) : ('a -> 'b) -> ('c -> 'a) -> 'c -> 'b
+(** Function composition: [f % g] is equivalent to [fun x -> f (g x)]. *)
```

Comment:
Could you please document the associativity and precedence of the operators here (as is now done for [the other operators](https://github.com/ocaml/ocaml/blob/63dbb11d4ddf72d7b143678790d20c4d1b2001e4/stdlib/stdlib.mli#L270)).

Reply:
Do you know where these 11 precedence levels come from?  How do they map to the table on http://caml.inria.fr/pub/docs/manual-ocaml/expr.html?

Just seeing that they are at the same level as ( *... ), I guess that would be 7/11 as well (and left-associative), right?

Reply:
They correspond to a subset of the table in the manual, starting with `:=` at level 1. See https://github.com/ocaml/ocaml/pull/1167.

Reply:
How do you map the 7/11 assigned to `(*)` to this table?

Reply:
Left-associative 7 is the right associativity and precedence. I should probably annotate the table with this operator level.

Reply:
Ok, thanks, got it. ( :: ) is listed in the table but is not one of those levels.  Annotating the table in the manual would definitely be useful.

Reply:
Info added to the docstrings.

Reply:
Note that function composition is associative, so the left/right associativity of the operator does not matter for these definitions.

Reply:
There are several operators at the same level (well, both % and %>, but many more), and one needs to know how they interact.